src/fragment-components/a-radiobutton-group.tsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import { Dropdown,
+	TextInput,
+	RadioButtonGroup } from 'carbon-components-react';
+import { AComponent, ComponentInfo } from './a-component';
+import { ComponentCssClassSelector } from '../components/css-class-selector';
+import image from './../assets/component-icons/radiobutton-group.svg';
+import {
+	angularClassNamesFromComponentObj,
+	nameStringToVariableString,
+	reactClassNamesFromComponentObj
+} from '../utils/fragment-tools';
+
+export const ARadioButtonGroupStyleUI = ({ selectedComponent, setComponent }: any) => {
+	const orientationItems = [
+		{id: 'horizontal', text: 'Horizontal'},
+		{id: 'vertical', text: 'Vertical'}
+	];
+
+	const labelPositions = [
+		{id: 'left', text: 'Left'},
+		{id: 'right', text: 'Right'}
+	];
+	selectedComponent.valueSelected = selectedComponent.defaultSelected;
+	return <>
+		<TextInput
+			value={selectedComponent.legend}
+			labelText='Legend name'
+			placeholder='Legend name'
+			onChange={(event: any) => {
+				setComponent({
+					...selectedComponent,
+					legend: event.currentTarget.value,
+				});
+			}}
+		/>
+		<Dropdown
+			label='Orientation'
+			titleText='Orientation'
+			items={orientationItems}
+			initialSelectedItem={orientationItems.find(item => item.id === selectedComponent.orientation)}
+			itemToString={(item: any) => (item ? item.text : '')}
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				orientation: event.selectedItem.id
+		})}/>
+		<Dropdown
+			label='Label position'
+			titleText='Label position'
+			items={labelPositions}
+			initialSelectedItem={labelPositions.find(item => item.id === selectedComponent.labelPosition)}
+			itemToString={(item: any) => (item ? item.text : '')}
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				labelPosition: event.selectedItem.id
+		})}/>
+		<ComponentCssClassSelector componentObj={selectedComponent} setComponent={setComponent} />
+	</>
+};
+
+export const ARadioButtonGroupCodeUI = ({ selectedComponent, setComponent }: any) => {
+	let allItems = selectedComponent.items.map((item: any) => {
+		return {
+			text: item.labelText,
+			id: item.id,
+			defaultChecked: item.defaultChecked
+		}
+	});
+	selectedComponent.defaultSelected = `${selectedComponent.items.find(((item: any) => {
+		return item.defaultChecked	
+	})).id}`;
+	
+	return <>
+		<Dropdown
+			label='Default selection'
+			titleText='Default selection'
+			items={allItems}
+			initialSelectedItem={allItems.find((item: any) => item.defaultChecked)}
+			itemToString={(item: any) => (item ? item.text : '')}
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				defaultSelected: `${event.selectedItem.id}`,
+				valueSelected:`${event.selectedItem.id}`,
```

Comment:
You're assigning the same values to defaultSelected & valueSelected so why have the code above on line 24? 
Instead, it makes sense to just have defaultSelected & completely delete valueSelected.

Have a look at https://github.com/carbon-design-system/carbon/blob/9407cb360ceb35ae28fe412e9bcf9af13a9c4650/packages/react/src/components/RadioButtonGroup/RadioButtonGroup.js#L85

Reply:
If ValueSelect is undefined then it uses defaultSelect,